src/ddb/ddb_main.c

```diff
@@ -74,93 +97,100 @@ run_cmd(struct ddb_ctx *ctx, struct argv_parsed *parse_args)
 		rc = ddb_run_rm_ilog(ctx, &info.dci_cmd_option.dci_rm_ilog);
 		break;
 	case DDB_CMD_CLEAR_CMT_DTX:
+		if (!write_mode) {
+			ddb_print(ctx, error_msg_write_mode_only);
+			rc = -DER_INVAL;
+			break;
+		}
 		rc = ddb_run_clear_cmt_dtx(ctx, &info.dci_cmd_option.dci_clear_cmt_dtx);
 		break;
 	}
 
-	ddb_str2argv_free(parse_args);
+	ddb_str2argv_free(&parse_args);
 
 	return rc;
 }
 
+static int
+process_line_cb(void *cb_args, char *line, uint32_t line_len)
+{
+	struct ddb_ctx *ctx = cb_args;
+
+	ddb_printf(ctx, "Command: %s", line);
+	if (line_len <= 1) /* It's okay to only have a '\n', but don't try to run a command */
```

Comment:
Since then, it is possible to contain empty lines, such as "\t\t\n".

Reply:
Good catch. Will resolve.